### ISSUE TITLE
Strange behavior to debug the application

### ISSUE THREAD

**Issue by joaom182** [node-inspector/node-inspector#589]
The node-inspector always stop at this line, instead of stopping at the place where I put the breakpoint.

![captura de tela 2015-03-19 as 16 50 15](https://cloud.githubusercontent.com/assets/3612098/6739725/7f6e12ec-ce59-11e4-8e47-7aaeb1f94e04.png)

I'm at Mac OSx Yosemite with node 0.12.0

**Comment by brian-mann** [comment]
Yup same problem here, I'm on node 0.12.0 and OSX 10.9.5

**Comment by 3y3** [comment]
Some small example of code?

**Comment by joaom182** [comment]
This is my server.js

```javascript
var http = require('http');
var domain = require('domain');
var d = domain.create();
var app = require('./app.js');
var path = require('path');
var load = require('express-load');
var port = process.env.PORT || 5002;
global.socket = null;

d.on('error', function(err) {
	console.error(err);
});

d.run(function() {
	var loader = load('extension-methods')
		.then('config')
		.then('storage')
		.then('services')
		.then('authentication')
		.then('authorization')
		.then('identity-servers')
		.then('enums')
		.then('events')
		.then('cache/manager.js')
		.then('cache/users.js')
		.then('models')
		.then('domain/services')
		.then('controllers')
		.then('routes')
		.then('tasks')
                .into(app, function() {
			app.listen(port);
			console.log('Running on localhost:' + port);
		});
});
```

**Comment by salomvary** [comment]
Same here, stops on the same line instead of the very first one. Debugger statements also don't work. Node 0.12.0.

**Comment by joaom182** [comment]
I installed nvm and the iojs via nvm, and now everything is back to normal operation.

**Comment by colingoldberg** [comment]
Same here. Node 0.12.0 and node-inspector 0.9.2 on MacOS

I'm getting used to using node with debug option ('node debug filename.js'). It's not as nice as node-inspector (when it works), but it works.

Colin G

**Comment by lyenliang** [comment]
I have the same problem on windows 7 with node-inspector 0.9.2 and nodw 0.12.0

**Comment by BenBBear** [comment]
#598

**Comment by Zoltu** [comment]
I'm experiencing the same problem when trying to debug my gulp tasks.  Breakpoint in chrome and `debugger;` line in code both result in breakpoints getting hit in other files (like gulp.js instead of my gulpfile.js).

Windows `8.1`
Node Inspector `0.10.0`
Node `0.12.1`
Gulp `3.8.11`

**Comment by 3y3** [comment]
https://github.com/node-inspector/node-inspector/pull/637

**Comment by 3y3** [comment]
Fixed in 0.10.1
